scanner/services/indexer.go

```diff
@@ -114,11 +128,31 @@ func validateContainerImageRequest(req *v4.CreateIndexReportRequest) error {
 	return nil
 }
 
-func (s *indexerService) GetIndexReport(ctx context.Context, req *v4.GetIndexReportRequest) (*v4.IndexReport, error) {
-	return nil, status.Error(codes.Unimplemented, "method GetIndexReport not implemented")
+// createManifestDigest creates a unique claircore.Digest from a Scanner's manifest hash ID.
+func createManifestDigest(hashID string) (claircore.Digest, error) {
```

Comment:
Very minor, but maybe check if hashID is empty

Reply:
good point, do we verify anywhere that hashID is something we'd consider valid?